security/keycloak/src/test/java/io/quarkus/ts/openshift/security/keycloak/SecurityKeycloakOpenShiftIT.java

```diff
@@ -31,6 +31,7 @@
 @AdditionalResources("classpath:keycloak.yaml")
 public class SecurityKeycloakOpenShiftIT {
     private static String getKeycloakUrl(OpenShiftClient oc) {
+        // @BeforeApplicationDeployment method is invoked too early for @TestResource @WithName("keycloak-plain") approach
```

Comment:
Actually, now that I'm thinking about it, it should be possible to add `URL` as a parameter of the `@BeforeApplicationDeployment` method. I'll merge this PR, but we should try if that works. (Obviously it won't work for the application, but `@AdditionalResources` are deployed by the time `@BeforeApplicationDeployment` runs.)

Wanna try it now, or should I merge and we'll try later?

Reply:
I will try later. Meetings marathon now

Reply:
Please merge ...